staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook_test.go

```diff
@@ -91,15 +88,11 @@ func (t *testWebhookHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.Error(w, err.Error(), http.StatusInternalServerError)
 }
 
-func newTestBlockingWebhook(t *testing.T, endpoint string, groupVersion schema.GroupVersion) *blockingBackend {
-	return newWebhook(t, endpoint, ModeBlocking, groupVersion).(*blockingBackend)
+func newTestBlockingWebhook(t *testing.T, endpoint string, groupVersion schema.GroupVersion) *backend {
```

Comment:
`newTestBlockingWebhook` -> `newTestWebhookBackend` or even `newTestBackend`

Reply:
We should just get rid of one of the methods, probably the next one

Reply:
newTestBackend SGTM